packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTag.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react';
+import { X } from '@phosphor-icons/react';
+
+import { CWCommunityAvatar } from '../cw_community_avatar';
+import { CWText } from '../cw_text';
+import { getClasses } from '../helpers';
+import { ComponentType } from '../types';
+
+import 'components/component_kit/new_designs/CWTag.scss';
+
+type TagProps = {
+  communityName: string;
+  disabled?: boolean;
+  onClick: () => void;
+};
+
+export const CWTag: FC<TagProps> = ({ communityName, disabled, onClick }) => {
+  const handleClick = () => {
+    onClick();
+  };
+
+  return (
+    <div className="SearchBarTag">
+      <div className="name">
+        <CWCommunityAvatar size="small" community={null} />
```

Comment:
how I am supposed to display the community avatar from outside of the component? this should not be `null` -- it should be a prop passed from the parent, similarly to `communityName`

Reply:
Addressed.